app/controllers/buskers_controller.rb

```diff
@@ -1,5 +1,22 @@
+# == Schema Information
+#
+# Table name: buskers
+#
+#  id          :integer          not null, primary key
+#  name        :string
+#  description :text
+#  image_id    :string
+#  facebook    :string
+#  twitter     :string
+#  website     :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
 class BuskersController < ApplicationController
   before_action :set_busker, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!,     only: [:create, :new, :edit, :update, :destroy]
```

Comment:
You can flip this around and say `except: [:show]` which is shorter and is then a touch safer as new actions you may add are assumed to be auth'd rather than public.